src/components/CheckboxGroup/CheckboxGroup.jsx

```diff
@@ -12,12 +12,12 @@ import dispatchAnalyticsEvent from '../../helpers/analytics';
  */
 class CheckboxGroup extends React.Component {
   UNSAFE_componentWillMount() {
-    this.inputId = this.props.id || _.uniqueId('errorable-checkbox-buttons-');
+    this.inputId = this.props.id || uniqueId('errorable-checkbox-buttons-');
   }
 
   getMatchingSubSection = (checked, optionValues) => {
     if (checked && this.props.children) {
-      const children = _.isArray(this.props.children)
+      const children = isArray(this.props.children)
```

Comment:
We can skip the extra import and use a native function for this:

```suggestion
      const children = Array.isArray(this.props.children)
```

https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_isarray

Reply:
Good idea, I'll update this in the other files.